clients/src/test/java/org/apache/kafka/clients/admin/AdminClientTestUtils.java

```diff
@@ -33,6 +27,12 @@
 import org.apache.kafka.common.config.ConfigResource;
 import org.apache.kafka.common.internals.KafkaFutureImpl;
 
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Map;
+import java.util.stream.Collectors;
+
```

Comment:
Let's revert this please as it is unrelated.

Reply:
Oh sure, not intentional (unwanted IDE setting, fixed)